### PR TITLE
Docs: Authentication

### PR DESCRIPTION
**Pull request by mendonk**
1. Confirmed values are still correct.
2. Walk user through creating a superuser, then adding users to their deployment, so they understand how this authentication is being used.
3. Clean up tables and explanations.

### CODE REVIEW

File: docs/docs/Configuration/configuration-authentication.md [hunk 1]
```diff
@@ -4,149 +4,117 @@ sidebar_position: 0
...
+| `LANGFLOW_AUTO_LOGIN` | Enables automatic login | `True` |
+| `LANGFLOW_SUPERUSER` | Superuser username | - |
+| `LANGFLOW_SUPERUSER_PASSWORD` | Superuser password | - |
+| `LANGFLOW_SECRET_KEY` | Key for encrypting superuser password | - |
+| `LANGFLOW_NEW_USER_IS_ACTIVE` | Automatically activates new users | `False` |
 
-By default, this variable is set to `True`. When enabled (`True`), Langflow operates as it did in versions prior to 0.5—automatic login without requiring explicit user authentication.
+### LANGFLOW_AUTO_LOGIN
 
+By default, this variable is set to `True`. When enabled, Langflow operates as it did in versions prior to 0.5—automatic login without requiring explicit user authentication.
```
Comment (KimberlyFields) [inline docs/docs/Configuration/configuration-authentication.md:86]:
```suggestion
By default, this variable is set to `True`. When enabled, Langflow operates as it did in versions prior to 0.5, including automatic login without requiring explicit user authentication.
```

**Review by KimberlyFields — APPROVED** [review]
Made one small suggestion, but approving.

File: docs/docs/Configuration/configuration-authentication.md [hunk 2]
```diff
@@ -4,149 +4,117 @@ sidebar_position: 0
...
 
+1. Create a `.env` file and open it in your preferred editor.
 
-The login functionality in Langflow serves to authenticate users and protect sensitive routes in the application. Starting from version 0.5, Langflow introduces an enhanced login mechanism that is governed by a few environment variables. This allows new secure features.
+2. Add the following environment variables to your file.
 
+```bash
+export LANGFLOW_AUTO_LOGIN=False
+export LANGFLOW_SUPERUSER=admin
+export LANGFLOW_SUPERUSER_PASSWORD=securepassword
```
Comment (viniciossilva3) [inline docs/docs/Configuration/configuration-authentication.md:22]: The default user here was:
```
username: langflow
password: langflow
```

Adding the following variables didn't work to create the admin user, you need to manually create a username and password and then use the default account to give it admin permissions::
```
LANGFLOW_SUPERUSER=admin
LANGFLOW_SUPERUSER_PASSWORD=securepassword
```

Autologin is not working when adding the variable as false:
`LANGFLOW_AUTO_LOGIN=False`

Adding the variable for new users to be active also did not work:
`LANGFLOW_NEW_USER_IS_ACTIVE=False`

I will report this to see if it is a bug.

I have not been able to test this yet but I will check:
`LANGFLOW_SECRET_KEY=randomly_generated_secure_key`

File: docs/docs/Configuration/configuration-authentication.md [hunk 3]
```diff
@@ -4,149 +4,117 @@ sidebar_position: 0
...
 
- It is critical not to expose these environment variables in your code repository. Always set them securely in your deployment environment, for example, using Docker secrets, Kubernetes ConfigMaps/Secrets, or dedicated secure environment configuration systems like AWS Secrets Manager.
+Langflow provides a command-line utility for interactively creating superusers:
 
+1. Enter the CLI command:
 
-:::
+```bash
+langflow superuser
+```
```
Comment (viniciossilva3) [inline docs/docs/Configuration/configuration-authentication.md:59]: That worked!

File: docs/docs/Configuration/configuration-authentication.md (hunk 1 above)
Comment (mendonk) [reply docs/docs/Configuration/configuration-authentication.md]: Added

**Pull request merged**